File: Mixter/Domain/Subscriptions/UserFollowed.cs
namespace Mixter.Domain.Subscriptions
{
    public struct UserFollowed : IDomainEvent
    {
        public SubscriptionId SubscriptionId { get; private set; }

        public UserFollowed(SubscriptionId subscriptionId)
            : this()
        {
            SubscriptionId = subscriptionId;
        }

        public object GetAggregateId()
        {
            return SubscriptionId;
        }
    }
}

File: app/Domain/Subscriptions/UserFollowed.php
<?php

namespace App\Domain\Subscriptions;

use App\Domain\IDomainEvent;

class UserFollowed implements IDomainEvent
{
    /**
     * @var SubscriptionId
     */
    private $subscriptionId;

    public function __construct(SubscriptionId $subscriptionId)
    {
        $this->subscriptionId = $subscriptionId;
    }

    /**
     * @return SubscriptionId
     */
    public function getSubscriptionId()
    {
        return $this->subscriptionId;
    }

    /**
     * @return string
     */
    public function getAggregateId()
    {
        return $this->getSubscriptionId()->getFolloweeId()->getId().'_'.$this->getSubscriptionId()->getFollowerId()->getId();
    }
}